cpp/lib/src/SysLog.cc

```diff
@@ -0,0 +1,38 @@
+/** \file    SysLog.h
+ *  \brief   Declaration of class SysLog.
```

Comment:
I guess this should be "Implementation" and not "Declaration", which is also redundant because we know it due to the file extension...